### PR TITLE
Release 1.15.1

### PR DESCRIPTION
**Pull request by rajaths010494**
Update setup.cfg, docs/conf.py and CHANGELOG.rst with the changes pertaining to release 1.15.1

### CODE REVIEW

File: CHANGELOG.rst [hunk 1]
```diff
@@ -1,6 +1,28 @@
...
 =========
 
+
+1.15.1 (2023-03-09)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Support retriveing extra fields which do not have extra prefix set in ``AzureDataFactoryHookAsync``.
```
Comment (phanikumv) [inline CHANGELOG.rst:11]: this needs to be changed similar to the airflow PR description

File: CHANGELOG.rst [hunk 2]
```diff
@@ -1,6 +1,28 @@
...
+  (`#899 <https://github.com/astronomer/astronomer-providers/pull/899>`_)
+- Pass connection_id in ``HttpTrigger``.
+  (`#896 <https://github.com/astronomer/astronomer-providers/pull/896>`_)
+- Fix ``SFTPSensorAsync`` using newer_than doesn't work well with file_pattern and
```
Comment (phanikumv) [inline CHANGELOG.rst:15]: needs to be a user facing description

File: CHANGELOG.rst [hunk 3]
```diff
@@ -1,6 +1,28 @@
...
+  set ``SFTPHookAsync`` default_known_hosts to None.
+  (`#905 <https://github.com/astronomer/astronomer-providers/pull/905>`_)
+- Make ``SageMakerTransformOperatorAsync`` and ``SageMakerTrainingOperatorAsync`` compatible with ``apache-airflow-providers-amazon>=7.3.0``.
+  (`#912 <https://github.com/astronomer/astronomer-providers/pull/912>`_)
+- Fix ``BigQueryInsertJobOperatorAsync`` to send location parameter to hook.
+  (`#866 <https://github.com/astronomer/astronomer-providers/pull/866>`_)
+- Update ``ExternalTaskSensorAsync`` and ``SnowflakeSensorAsync`` to check on worker first
```
Comment (phanikumv) [inline CHANGELOG.rst:22]: what's `check on worker first` ? please elaborate

**Review by phanikumv — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 1 above)
Comment (rajaths010494) [reply CHANGELOG.rst]: Changed

File: CHANGELOG.rst (hunk 3 above)
Comment (rajaths010494) [reply CHANGELOG.rst:22]: changed

File: CHANGELOG.rst [hunk 4]
```diff
@@ -1,6 +1,28 @@
...
+
+1.15.1 (2023-03-09)
+-------------------
+
+Bug Fixes
+"""""""""
+
+- Allow and prefer non-prefixed extra fields for ``AzureDataFactoryHookAsync``.
+  (`#899 <https://github.com/astronomer/astronomer-providers/pull/899>`_)
+- Pass connection_id in ``HttpTrigger``.
```
Comment (phanikumv) [inline CHANGELOG.rst:13]: Fix `HttpSensorAsync` to use the correct connection id instead of a default connection id

File: CHANGELOG.rst (hunk 2 above)
Comment (phanikumv) [reply CHANGELOG.rst:15]: can you elaborate on what you mean by not working well with file_pattern

File: CHANGELOG.rst (hunk 3 above)
Comment (phanikumv) [reply CHANGELOG.rst:22]: Mitigate race condition on deferrable sensors ``ExternalTaskSensorAsync`` and ``SnowflakeSensorAsync`` immediately getting successful, by making the changes to these sensors to check on the worker first, then defer if the condition is not met.

**Review by phanikumv — CHANGES_REQUESTED** [review]

File: CHANGELOG.rst (hunk 2 above)
Comment (rajaths010494) [reply CHANGELOG.rst]: added details.

File: CHANGELOG.rst (hunk 3 above)
Comment (rajaths010494) [reply CHANGELOG.rst]: added above descritpion

File: CHANGELOG.rst (hunk 4 above)
Comment (rajaths010494) [reply CHANGELOG.rst]: added it.

**Review by phanikumv — APPROVED** [review]

**Pull request merged**